Servers/WindowServer/WSWindowManager.cpp

```diff
@@ -196,6 +196,7 @@ const Font& WSWindowManager::app_menu_font() const
 void WSWindowManager::set_resolution(int width, int height)
 {
     WSCompositor::the().set_resolution(width, height);
+    m_menu_manager.set_dirty();
```

Comment:
"dirty" is very vague. Let's be more expressive :)
We could either just tell the `WSMenuManager` that a resolution change happened, and let it deal with it internally.
```cpp
m_menu_manager.did_change_resolution(width, height);
```
Or we could tell it to resize the window:
```cpp
m_menu_manager.set_needs_window_resize();
```